third_party/maya/lib/usdMaya/MayaTransformWriter.cpp

```diff
@@ -565,9 +598,8 @@ bool MayaTransformWriter::writeTransformAttrs(
     // Write parent class attrs
     writePrimAttrs(mXformDagPath, usdTime, xformSchema); // for the shape
 
-    // can this use xformSchema instead?  do we even need _usdXform?
-    computeXFormOps(xformSchema, mAnimChanList, usdTime,
-                    _GetSparseValueWriter());
+    computeXFormOps(xformSchema, mAnimChanList, usdTime, getArgs().eulerFilter,
+            previousRotates, _GetSparseValueWriter());
```

Comment:
I've resolved the merge conflicts in this PR locally, but my builds are failing because this ```previousRotates``` parameter isn't defined in this function. Where should this be coming from?